packages/storybook/src/utils/IncludeDocumentation.tsx

```diff
@@ -0,0 +1,25 @@
+import { Markdown } from '@storybook/blocks';
+import { ReactElement } from 'react';
+
+interface DocumentationProps {
+  component: string;
+  url: string;
+  markdown: string;
+}
+
+export const IncludeDocumentation = ({ component, url, markdown }: DocumentationProps): ReactElement => {
+  return (
+    <>
+      <p className="lsb-citation">
+        {`Documentatie afkomstig van `}
+        <cite>
+          {/* prettier-ignore */}
+          <a href={url}>{component}</a>
+        </cite>
+        {`.`}
+      </p>
+
+      <Markdown>{markdown}</Markdown>
```

Comment:
Dit mag er wel uit. Als we straks meer met microfrontends gaan doen gaat dit in de web zitten

Reply:
Nog niet alle componenten hebben micro-content.
Ik verwacht dat we binnenkort een mix van oude Readme's en micro-content hebben.
Dan bestaat deze naast de micro-content-versie.